app/app.go

```diff
@@ -183,6 +184,10 @@ func addData(management *config.ManagementContext, cfg Config) error {
 		return err
 	}
 
+	if err := addFeatures(cfg); err != nil {
```

Comment:
nit use `=` here instead of `:=`

Reply:
Although there is an existing err scoped above, the value for err has no reason to outlive the block and this conforms to how the rest of the error assignments in the function.